com.maplerain.telsis_translator_flutter.yml

```diff
@@ -0,0 +1,34 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+---
+app-id: com.maplerain.telsis_translator_flutter
```

Comment:
`maplerain.com` doesn't exist on wherever it is being hosted. Is this domain really controlled by the application developers?

Reply:
I own the domain name "maplerain.com" and my main business site is at [www.maplerain.com](https://www.maplerain.com).